### ISSUE TITLE
🚑 npm registry trouble for 1.0.0-beta.2, 1.0.0-beta.3

### ISSUE THREAD

**Issue by kyuwoo-choi** [gs-shop/vue-slick-carousel#51]
```
error https://registry.yarnpkg.com/vue-slick-carousel/-/vue-slick-carousel-1.0.0-beta.2.tgz: Extracting tar content of undefined failed, the file appears to be corrupt: "Unexpected end of data"
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

![image](https://user-images.githubusercontent.com/1215767/70915628-82feb780-205d-11ea-89ac-fe33ef77ca9a.png)

wait for recovery.

**Comment by kyuwoo-choi** [comment]
https://github.com/npm/cli/issues/602

**Comment by kyuwoo-choi** [comment]
use `1.0.0-beta.1` at this moment.

**Comment by kyuwoo-choi** [comment]
Issue seems resolved, able to fetch version 1.0.0-beta.3

![image](https://user-images.githubusercontent.com/1215767/70920256-9e6dc080-2065-11ea-9a4f-d88062aa71bb.png)

use `1.0.0-beta.3`
